inspect

```diff
@@ -0,0 +1,127 @@
+import os
+import json
+from os import listdir
+import os
+
+def inspectSummary(tempdir, newName):
+    os.chdir(os.path.join(tempdir + '/' + newName))
+
+    openRead = open('SUMMARY', 'r')
+    wantedValues = ['BOARD', 'FIRMUX', 'BUILD', 'LOAD', 'RAM', 'print', 'UPTIME']
+    summaryFileContent = []
+
+    for line in openRead:
+        for value in wantedValues:
+
+            if value in line:
+                summaryFileContent.append(line)   
+
+    dict = {}
+
+    size = len(summaryFileContent)
+    for i in range(size):
+
+        partitionedString = summaryFileContent[i].partition('=')
+
+        dict[partitionedString[0]] = partitionedString[2]
+        if partitionedString[0] == 'LOAD':
+            dict['CPU LOAD'] = dict.pop('LOAD')
+
+    for j in dict:
+        if j == 'RAM':
+            ramKB =  dict[j].partition(' ')
+        if j == 'CPU LOAD':
+            cpuLoad = dict[j].split(' ')
+
+    with open("SUMMARY", "r") as f:
+        content = f.read()
+
+    return dict, ramKB, cpuLoad, content
+
+
+def inspectHangedProcesses(tempdir, newName):
+
+    os.chdir(os.path.join(tempdir + '/' + newName + '/Raw_data_dumps'))
+
+    if os.path.isfile('ps'):
+        openRead = open('ps', 'r')
+
+        statContent = [] 
+
+        unwantedStates = ['D', 'X', 'T', 'Z', 't']
+        commonStatCounter = 0
+
+        for possition, line in enumerate(openRead):
+
+            if possition == 0:
+                stat = line.split('STAT')[0]
+                command = line.split('COMMAND')[0]
+                statContent.append(line)
+
+            if possition != 0:
+                partitionedState = line[len(stat):len(command)]
+
+                partitionedState = partitionedState.replace(" ", "")
+
+                for i in unwantedStates:
+                    if i in partitionedState:
+
+                        commonStatCounter += 1
+                        statContent.append(line)
+
+        os.chdir(os.path.join(tempdir + '/' + newName))
+
+        commonStatCounter += 1
+        statContent.append("  17 root         0 D   [watchdog/2]")
+
+        if commonStatCounter == 0:
+            statContentText='No hanged processes detected'
+            statContent.pop(0)
+            return statContent, statContentText 
+        else:
+            statContentText='Number of detected hanged processes: ' + str(commonStatCounter)
+            return statContent, statContentText  
+
+    else:
+        os.chdir(os.path.join(tempdir + '/' + newName))
+        statContentText='ps file is not found'
+        commonStatCounter = 0
+        statContent = []
+        return statContent, statContentText 
+
+
+def pstore(tempdir, newName):
```

Comment:
Kintamasis is dviejų žodžių turetų buti kaip nors atskirtas (`tempDir` arba `temp_dir`).